bigframes/ml/llm.py

```diff
@@ -598,6 +598,8 @@ class TextEmbeddingGenerator(base.BaseEstimator):
         connection_name (str or None):
             Connection to connect with remote service. str of the format <PROJECT_NUMBER/PROJECT_ID>.<LOCATION>.<CONNECTION_ID>.
             If None, use default connection in session context.
+        row_limit (int or None):
+            The maximum number of rows that this model can process.
```

Comment:
nit: `The maximum number of rows that this model can process per call`. The model can be called multiple times.

Reply:
SG!